package.json

```diff
@@ -66,7 +66,8 @@
     "node": ">=6.0.0"
   },
   "resolutions": {
-    "natives": "1.1.3"
+    "natives": "1.1.6",
+    "node-sass": "4.5.3"
```

Comment:
Pinned node-sass to an earlier version as later versions have changed/break how CSS variables are expanded.